sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1085,3 +1074,25 @@ func withDynamicResolver(ctx context.Context, ref string, registryConfig *Regist
 		return defaultResolver
 	}
 }
+
+// withMaskedPaths sets an alternate list of masked paths
+func withMaskedPaths() oci.SpecOpts {
+	return func(_ context.Context, _ oci.Client, _ *containers.Container, s *runtimespec.Spec) error {
+		if s.Linux != nil && s.Linux.MaskedPaths != nil {
+			// Paths we want to unmask
+			unmaskPaths := []string{
+				"/sys/firmware",
```

Comment:
Can we add a comment on why we want to unmask this? Just something to remind us why we went through the work.